OWNERS

```diff
@@ -12,4 +12,12 @@ reviewers:
   - vimalk78
   - syedriko
   - periklis
+  - vparfonov
+  - eranra
+  - pmoogi-redhat
+  - ajaygupta978
 component: "Logging"
+filters:
+  "Dockerfile":   # matches Dockerfile, Dockerfile.rhel8, bundle.Dockerfile
```

Comment:
Does it have to be a proper regex?

Reply:
it has to be a Go regex. I tested it like
```
func test_dockerfiles() {
	re := regexp.MustCompile(`Dockerfile`)
	fmt.Println(re.String())
	// true
	fmt.Println("-- trues below ---")
	fmt.Println(match(re, "Dockerfile"))
	fmt.Println(match(re, "Dockerfile."))
	fmt.Println(match(re, "Dockerfile.rhel8"))
	fmt.Println(match(re, "bundle.Dockerfile"))
	// false
	fmt.Println("-- falses below ---")
	fmt.Println(match(re, "dockerfile"))
}
```